CMakeLists.txt

```diff
@@ -58,19 +58,19 @@ IF(FXDIV_BUILD_TESTS)
     SET(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
     ADD_SUBDIRECTORY(
       "${GOOGLETEST_SOURCE_DIR}"
-      "${CMAKE_BINARY_DIR}/googletest")
+      "${CMAKE_BINARY_DIR}/googletest")/C
```

Comment:
This line introduces an error for me. This is the output from cmake 3.29.2 for fda05f07818d99f774592fcde9b24547207bf3c0 applied to https://github.com/Maratyszcza/FXdiv/commit/63058eff77e11aa15bf531df5dd34395ec3017c8:
```
CMake Error at CMakeLists.txt:61:
  Parse error.  Expected a command name, got unquoted argument with text
  "/C".


-- Configuring incomplete, errors occurred!
```
Removing this line allows building with 1.14.0.

Reply:
Fixed. That change was not intended to be done